CHANGELOG.md

```diff
@@ -32,6 +32,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   status data.
 ### Changed
 - Updated the README file to remove outdated information.
+- Changed Jobstat error structure and codes 
```

Comment:
It's all right to skip adding a change log entry for this pull request, since this is modifying an existing feature branch.